packages/shared/src/styles/base.css

```diff
@@ -54,12 +54,14 @@ body {
   --theme-status-help: theme('colors.cheese.40');
   --theme-status-success: theme('colors.avocado.40');
   --theme-status-warning: theme('colors.bun.40');
-  --theme-status-cabbage: theme('colors.cabbage.40'); /* temporary */
+  --theme-status-cabbage: theme('colors.cabbage.40');
   --theme-status-fill: theme('colors.water.40');
 
   --theme-shadow2: theme('boxShadow.2-black');
   --theme-shadow3: theme('boxShadow.3-black');
 
+  --theme-mention: rgba(206, 61, 243, 0.24);
+  --theme-mention-hover: rgba(206, 61, 243, 0.32);
```

Comment:
Unfortunately, the value like `theme('colors.pepper.70')66` (with 66 to provide opacity percentage on hex values) doesn't provide the accurate color of the requirements. We will have to use the RGB equivalent but it is not being used in our configuration so I had to specifically get the cabbage color specs. Pseudo-elements have been proven to result in edge cases.